include/aspect/material_model/reactive_fluid_transport.h

```diff
@@ -60,6 +60,13 @@ namespace aspect
          * @}
          */
 
+        double compute_bulk_density (const double porosity,
+                                     const double solid_density,
+                                     const double fluid_density) const;
+
+        double compute_mass_fraction (const double porosity,
+                                      const double solid_density,
+                                      const double fluid_density) const;
```

Comment:
Please add documentation, in particular which mass fraction you compute, the one of the fluid? the one of the solid? and why are the parameters named differently in the .cc file from here?